resinhup.py

```diff
@@ -132,7 +132,7 @@ def main():
         images_fingerprint_path = os.path.join(os.path.dirname(sys.modules[__name__].__file__), "modules/fingerprint/known-images")
 
         # Make sure the boot partition device is mounted
-        bootdevice = getBootPartition(self.conf)
+        bootdevice = getBootPartition(args.conf)
```

Comment:
Can this commit be incorporated in the 5th commit where this change is introduced? Since this is PR level, we can have clean commits merged in the git tree rather than introduce fixes commits like this one